test/directoryAsModule.js

```diff
@@ -0,0 +1,8 @@
+var assert = require('assert');
+var requireDir = require('..');
+
+assert.deepEqual(requireDir('./directoryAsModule'), {
+    a: 'a', b: 'b'
```

Comment:
Minor: let's keep to the code style of having these object keys be on their own lines.